flytekit/image_spec/image_spec.py

```diff
@@ -50,6 +50,7 @@ class ImageSpec:
         commands: Command to run during the building process
         tag_format: Custom string format for image tag. The ImageSpec hash passed in as `spec_hash`. For example,
             to add a "dev" suffix to the image tag, set `tag_format="{spec_hash}-dev"`
+        copy_src_dest: List of tuples containing source files/directories and their corresponding destination directory for copying.
```

Comment:
When we set `source_root`, it copies everything into `/root`, should we make that assumption here too? Then this just becomes a list of files to copy.

Reply:
If copying files only to `/root` meets our needs, then managing a single list of files will be sufficient. Does this approach meet our needs?